src/components/OpenFileDialog/DateField.js

```diff
@@ -0,0 +1,17 @@
+import PropTypes from 'prop-types'
+import React from 'react'
+
+export const DateField = ({ date }) => {
+    const d = new Date(date)
+    const time = `${d.getFullYear()}-${('0' + (d.getMonth() + 1)).slice(-2)}-${(
```

Comment:
Date formats are always tricky since they're different between regions. Should we use a localised date string here instead?

Reply:
Yes, that would definitely be better.
I just copied over from the old component.
I guess we need to add a dependency for this.
I remember I looked into https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat but I don't remember why I didn't end up using it...

Reply:
Yeah, I guess this is generic thing for all our apps. Maybe check in #frontend if someone has a generic solution for date string formatting and localisation?

Reply:
I tried using `moment` but `en` by default uses the American format, which is then different from the current default.
Although is technically more correct, I'm wondering if this can be a problem.
We don't seem to have the possibility to choose the variant of English in the user settings, so just by having `en` `moment` seem to default to the American variant.

Reply:
Fixed using `moment`.
As mentioned above, this has the effect that for `en` the date format is the American one, which is different compared to what we had before, which was a fixed format `yyyy-mm-dd`